content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  A tutorial on how to setup your own self-hosted runners with GPUs using CML cloud-runners in your CML workflow in Github or Gitlab.
+picture: 2020-08-05/header.png
+author: david_g_ortega
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/??
+tags:
+  - CML
+  - GPUs
+  - Self-hosted runner
+  - Continuous Machine Learning
+  - Github
+  - Gitlab
+  - Technical debt
+  - Tutorial
```

Comment:
Probably too many tags (and too specific ones). You can check https://dvc.org/blog/tags/{tag} to see if there's any content there already.

Not sure what our blog post tagging guidelines are actually.

Reply:
BTW @elleobrien https://dvc.org/blog/tags/CML has no content, may want to add that one to some prior blog posts? 🙂

Reply:
@jorgeorpinel good idea!